scripts/apptainer_generator.py

```diff
@@ -195,6 +197,10 @@ def apptainer_pull(self, project: str, name: str, tag: str, args=None):
         command = ['apptainer', 'pull']
         if args is not None:
             command += args
+        if (hasattr(self.args, 'disable_cache') and
+            self.args.disable_cache and
+            '--disable-cache' not in command):
```

Comment:
One question re: this logic - where else might the attribute disable_cache be set, other than the argument flag you have added? Seems like we might not need the explicit `--disable-cache` condition here.

Reply:
Essentially, why are we setting this up to add the flag if adding the flag sets these others?

Reply:
Hmm....I suppose this sets up the passthrough of this flag to the other apptainer commands then.....I think I get it.

Reply:
Are we good? 😆 can talk in person.